openqa_bugfetcher/issues/bugzilla_issue.py

```diff
@@ -27,8 +28,11 @@ def rest_get_bug(issue_id):
             req = rest_get_bug(issue_id)
             assert req.status_code != 401, "Wrong auth for Bugzilla"
             assert req.status_code != 403, "Insufficient permission to access this bug (or the API at all)"
-            assert req.ok, f"Server Status != 200: {req.text}"
-            data = req.json()
+            assert req.ok, f"Server Status != 2xx: {req.text!r}"
+            try:
+                data = req.json()
+            except json.decoder.JSONDecodeError:
+                raise AssertionError(f"No valid JSON returned from server: {req.text!r}")
```

Comment:
why do you want to turn the JSONDecodeError into a more generalized AssertionError? I don't think AssertionError should be used for this. How about just catching "json.decoder.JSONDecodeError" in bin/fetch_openqa_bugs same as you do for requests.exceptions.ReadTimeout already?